index.bs

```diff
@@ -12,6 +12,12 @@ Abstract: TODO
 Group: WebView-CG
 </pre>
 
+# Abstract # {#abstract}
+
+This document defines an API for embedding arbitrary web content only within the
+context of a Isolated Web Applications (IWAs). The embedded content is a new
+top-level browsing context that is able to be controlled by the API.
```

Comment:
In future changes I think it's best to collapse commits like this and the next one into a single commit in the PR. It will reduce reviewer load and your own churn if changes are made in the repo to this area prior to your PR landing.

Reply:
Would that be squashing these commits?

Reply:
Yes, that's squashing the commits.

Reply:
I'll resolve this since merging the PR will squash these commits, and I'll apply squashing to my future commits prior to starting a PR or before it's reviewed.